config/default/manager_auth_proxy_patch.yaml

```diff
@@ -10,7 +10,7 @@ spec:
     spec:
       containers:
       - name: kube-rbac-proxy
-        image: gcr.io/kubebuilder/kube-rbac-proxy:v0.5.0
+        image: kube-rbac-proxy:latest
```

Comment:
Let's keep this one as `v0.5.0` avoid using `latest` tag at all cost

Reply:
updated to `v0.5.0`, although we will not use this image and tag here, it should always be updated to the one in Makefile by kustomization.

I noticed one of my mistake, kustomization should change the RBAC image in `config/default` instead of `config/manager`, it is now tested and works as expected.

btw, I also noticed there is both `$(shell pwd)` and `$(PROJECT_DIR)` in the Makefile, it may cause confusion, so that I changed them to a uniq `$(PROJECT_DIR)`